frontend/src/views/Home.tsx

```diff
@@ -70,15 +71,10 @@ const Home: React.FC = () => {
           </SectionCard>
         </div>
         <div className="flex w-full flex-col gap-6 xl:w-1/2">
-          <SectionCard
-            title="Next Submission Deadline"
-            loading={nextTournament.isLoading}
-          >
-            {nextTournament.isSuccess && nextTournament.data !== null ? (
-              <CountdownDigital date={nextTournament.data.submission_freeze} />
-            ) : (
-              "No upcoming submission deadlines."
-            )}
+        <SectionCard title="Next Submission Deadline" loading={nextTournament.isLoading || episode.isLoading}>
+            {!isNil(episode.data) && new Date().getTime()>episode.data.game_release.getTime() ?
```

Comment:
I think we can have separate section cards for each of these countdowns? and then we don't need these nested ternaries :)